sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -483,6 +495,91 @@ _nib_offl_entry_t *_nib_offl_iter(const _nib_offl_entry_t *last)
     return NULL;
 }
 
+bool _nib_offl_is_entry(const _nib_offl_entry_t *entry)
+{
+    return (entry >= _dsts) && _in_dsts(entry);
```

Comment:
isn't `_in_dsts()` also suppoed to check for >= `_dsts`?

Reply:
you could probably remove `_nib_offl_is_entry` and use `_in_dsts()` everywhere

Reply:
Nope. Since it is used as a loop-target this check wouldn't make sense / would be more expensive

Reply:
But I can do the replacement for the cases where `entry >= _dsts` must also be true ;-).

Reply:
(there is no such case in `_nib-internal.c` so all done ;-))